docs/docs/getting-started/glossary.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Glossary
+sidebar_label: Glossary
+sidebar_position: 3
+---
+
+This glossary provides concise definitions for some of the key concepts within Flagsmith:
+
+- [**A/B Testing**](/advanced-use/ab-testing.md): A method of testing different feature variants with different user groups, often implemented using multivariate flags and percentage splits.
+
+- [**Core API**](/edge-api/Overview.md#core-api): Flagsmith's private API for programmatic control of the platform.
+
+- [**Edge API / Edge Proxy**](/advanced-use/edge-api.md): Flagsmith's publicly accessible API, specifically intended for use with our SDKs.
+
+- [**Environment**](/basic-features/index.md#environments): Environments are a way to separate the configuration of your features. A project can have any number of environments.
+
+- [**Environment Document**](/clients/index.md#the-environment-document): A JSON document containing all configuration for feature flags in an environment.
+
+- [**Feature**](/basic-features/managing-features.md): A configuration that can be enabled, disabled, or set to a specific value. Features are shared across all Environments in a project, but their values/states can be modified between Environments.
+
+- [**Feature Flag**](/basic-features/index.md): A boolean or multivariate switch to enable/disable features or set their values dynamically without deploying code.
+
+- [**Identity**](/basic-features/managing-identities.md): An identity represents a specific user within a particular environment. Identities allow you to manage and override feature settings for individual users, and the same user can have different features enabled in different environments.
```

Comment:
Identities need not be a 'user' - they can be anything (it's why they're called identities, and not users). Let's be careful not to refer to identities as users please.